cli-rs/src/main.rs

```diff
@@ -62,13 +62,17 @@ enum Commands {
         #[arg(long)]
         coprocessors: Option<String>,
 
+        /// Convert from the executable ELF file instead of the assembly.
+        #[arg(short = 'e', long)]
+        #[arg(default_value_t = false)]
+        via_elf: bool,
```

Comment:
```suggestion
        elf: bool,
```
Looks nicer on the CLI to use `--elf`

Reply:
I was thinking into replacing this with `--asm`, and set ELF as default. This way it would see more usage and we could be more confident about removing the asm path.

Reply:
I have changed to ELF being the default.